gemfiles/rails_4.2.7.gemfile

```diff
@@ -0,0 +1,7 @@
+# This file was generated by Appraisal
+
+source "https://rubygems.org"
+
+gem "activerecord", "4.2.7"
+
+gemspec :path => "../"
```

Comment:
I guess we should use new Ruby syntax for those hashes, Rails 4.2.7 anyway requires modern Ruby